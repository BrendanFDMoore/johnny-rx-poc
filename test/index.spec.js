import 'index';
import { ActionTypes, XHR } from 'constants/index';

describe('Constants:ActionTypes', () => {
  it('should match the snapshot', () => {
    expect(ActionTypes).toMatchSnapshot();
  });
});

describe('Constants:XHR', () => {
  it('should match the snapshot', () => {
    expect(XHR).toMatchSnapshot();
  });
});
