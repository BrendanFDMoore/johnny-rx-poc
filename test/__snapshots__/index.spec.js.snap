// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Constants:ActionTypes should match the snapshot 1`] = `
Object {
  "CANCEL_FETCH": "CANCEL_FETCH",
  "FETCH_POPULAR_REPOS_CANCEL": "FETCH_POPULAR_REPOS_CANCEL",
  "FETCH_POPULAR_REPOS_FAILURE": "FETCH_POPULAR_REPOS_FAILURE",
  "FETCH_POPULAR_REPOS_REQUEST": "FETCH_POPULAR_REPOS_REQUEST",
  "FETCH_POPULAR_REPOS_SUCCESS": "FETCH_POPULAR_REPOS_SUCCESS",
  "HIDE_ALERT": "HIDE_ALERT",
  "SHOW_ALERT": "SHOW_ALERT",
  "USER_LOGIN_FAILURE": "USER_LOGIN_FAILURE",
  "USER_LOGIN_REQUEST": "USER_LOGIN_REQUEST",
  "USER_LOGIN_SUCCESS": "USER_LOGIN_SUCCESS",
  "USER_LOGOUT_FAILURE": "USER_LOGOUT_FAILURE",
  "USER_LOGOUT_REQUEST": "USER_LOGOUT_REQUEST",
  "USER_LOGOUT_SUCCESS": "USER_LOGOUT_SUCCESS",
}
`;

exports[`Constants:XHR should match the snapshot 1`] = `
Object {
  "FAIL": "FAIL",
  "SUCCESS": "SUCCESS",
}
`;
